OPUNetTransportLayer.cpp

```diff
@@ -1372,7 +1371,7 @@ void OPUNetTransportLayer::GetGameServerAddressString(char* gameServerAddressStr
 	// Get the address string
 	config.GetString(sectionName, "GameServerAddr", gameServerAddressString, maxLength, "");
 
-	logFile << "[" << sectionName << "]" << " GameServerAddr = " << gameServerAddressString << std::endl;
+	Log("GameServerAddr = " + std::string(gameServerAddressString));

```

Comment:
This dropped printing the .ini section name. Not sure if that was intentional or not. Might have some value in case an ini file has settings for multiple game servers. Probably not so useful in practice though.

Reply:
It was intentional on my part since the logger will list [NetFixClient.dll] [NetFix] MESSAGE if this is left in place. I can add it back in if you want it.

Reply:
Not sure if it matters too much. It does seem repetitive looking now, even though the two bits of information are different. It's probably fine to drop it. I think there's still sufficient information from the `gameServerAddressString` that people could figure out if the value was loaded from the wrong section.